src/rgw/motr/gc/gc.cc

```diff
@@ -112,6 +161,21 @@ int MotrGC::dequeue(const DoutPrefixProvider* dpp, std::string iname, motr_gc_ob
   return rc;
 }
 
+int MotrGC::get_locked_gc_index(uint32_t* rand_ind)
+{
+  int rc = -1;
+  uint32_t new_index = 0;
+  // attempt to lock GC starting with passed in index
+  for (int ind = 0; ind < max_indices; ind++)
```

Comment:
To introduce more randomness, we should start the loop with a random number and run in in cyclic manner.

Reply:
As discussed we do not need one more randomization